packages/cadl-ranch-specs/http/azure/core/lro/standard/main.tsp

```diff
@@ -165,9 +166,9 @@ Expected response body:
 }
 ```
 """)
-@global.Azure.Core.pollingOperation(ResourceOperations.GetResourceOperationStatus<User, never>)
```

Comment:
With this removal we should expect the status returned by `getLroMetadata` is not change?

Reply:
Yes. Currently (in 0.33.x) this `@pollingOperation` is ignored by `getLroMetadata` API, as it does not provide a parameter mapping.

In 0.34.0 or +, this `@pollingOperation` will have effect even without parameter mapping. See https://github.com/Azure/typespec-azure/issues/3410

Reply:
There could be slight change on `LroMetadata`. E.g. `logicalResult` maybe same properties, but it could be name="".

Reply:
Can you help me understand what is the `parameter mapping` used for?

I didn't notice there is any impact with these mapping included in [OpenAPI playground](https://cadlplayground.z22.web.core.windows.net/cadl-azure/?c=aW1wb3J0ICJAdHlwZXNwZWMvaHR0cCI7CtIZcmVzdNUZdmVyc2lvbmluZ8wfYXp1cmUtdG9vbHMvyCstxhVjb3JlIjsKCnVzaW5nIFR5cGVTcGVjLkh0dHA70BVSZXN00RVWyXLIG0HEWS5Db3Jl0hIuVHJhaXRzOwoKQHVzZUF1dGgoCiAgQXBpS2V5xA48xgtMb2NhdGlvbi5oZWFkZXIsICJhcGkta2V5Ij4gfCBPxCoyxS9bCiAgICB7xQYgIOQA3DrHH0Zsb3fkAKUuaW1wbGljaXQsxyVhdXRob3JpesVhVXJsOiAi5AFoczovL2xvZ2luLmNvbnRvc28uY29tL2NvbW1vbi9vxDUyL3YyLjAvyEBlIshSc2NvcGVzOiBbyUl3aWRnZXTNSi5kZWZhdWx0Il3GN30KICBdPgopCkBzZXJ2aWNlKOQAxnRpdGxlOiAiQ8Y6IFfFSSBNYW5hZ2VyIiwKfccxZXLkATkie2VuZHBvaW50fcd05QCV0D9BUElzxRnEYyAgQGRvYygiIiIKU3Vw5AIiZWTIKVPmAItzIMhXcyAocHJvdG9jb2wgYW5kIGhvc3RuYW1lLCBmb3IgZXhhbXBsZToK6QD4ZXN0dXMuYXBpLvIBAykuCiIiIinlAIPIYDogc3RyaW5n5ACdfeQBDecCwWVkKOcAxC7mAJvnAQLoAm9zKQrkAIdzcGFjZSDVKuQCY%2BUA6FRoySLvAU8g5wF6IMd1LiIpCmVudW0gyGXlATDGRscTIDIwMjItMDgtMzHlAMrkAsZEZXBlbmRlbmN5KOsC6chELnYxXzBfUHJldmlld18yxDVgyUUwYCwKfQoKLy8gTW9kZWxzIC%2FTAewAyWNvbG9yIG9mIGEg5wFh6AC05gDbQ8Ug6gC3QmxhY2vlAKrFCiwKyRpXaGl0ZcUaxQrLGlJlZMUYUmVkyxZHcmVlbsUYxQrLGkJsdcZJxAnlANDGGEHrAKNAcmVzb3VyY2UoIsYVcyIpCm3kAPToAZTlALRrZXnIIU5hbcZW6gD9xhkg5AH5xFQgIEB2aXNpYmlsaXR5KCJyZWHmALjEIOgCWDvqAKfLQuUBRsZDxQs67AE%2BzzNJROQBb3TJPSdzIG1hbnVmYWN0dXJlx0bMEklkzX0uLi5FdGFnUHJvcGVydHk76gEgxF5yZXBhaXIgc3RhdGXwAddAbHJvU3RhdHVz7AHiUsUyxBhl6wHoxhzHT3Mgc3VjY2VlZGVkxEggIFPID%2BwCAM4yZmFpbMgvRsUM2ix3ZXJlIGNhbmNlyTNDxw7SNXdhcyBzZW50IHRv5QFG8gE9U2VudFRvTcsY7QJFc3VibWl05ASl5wEtcmVxdWVzdOUEguwBMOwCT%2BYBJlLHKuoBKOQBdOkBbeoB1c9bxE4gyA3lAWvoAiLrAX7vAihkxFXkBSN0aW1lIHdoZW7FYMdZaXMgc2NoZWR1bGVk5AEKb2NjdecA%2F8kYRGF0ZVRp5AK8dXRjyA3fZslm5gDp5AFtY3JlYXToAZjHDd9b31vJW3VwxCXHW8cN31vfW8lbY2%2FkBkrJXckP11%2FtA0VwYXJhbWV0ZeQCx%2BsCFOUB6XVzyGf1AiLmA2BQxEHsBe70A%2FkgYmXkCMzmAdHoAKhAcGF0aAogxyPsA%2FvxBQcnc%2BQAqfAFDsQT5AUMQHBhcmVudFLIGeYApe4As1BhcvIFKMQV8wUs5AUF6ADAyHf%2FBTH0BTFJROQCrnVzZeUBWnJlb3JkZcQnz1jECu8E%2BPQBScQn%2FwVA%2FwVA8AVAZGV0YWlsc8RhYegAn%2B8EIeoBPPAEJcQUUsY08gQqSWRlbnRpZmllcyB3aG8gc2lnbmVkIG9m5gDKz2vnA8rFJE9mZkJ57QIILy8gQW7oCS3mAKvkCxdsZXRvbiDoAgvnANVQcm92aWRlcyBhbmFseXRpY3MgYWJvdXTFduQBiOQDR21haW50ZW7kBXTGT%2BYA0O8CW8lC%2FwJf6AEPQcly6gJkyURJ5gIr6gF6aegBIHLlAWjkAJ%2FJLSBvYmplY3QuICBUaOQGLGlzIG9ubHkgb27mAplkICdjdXLkAJEn%2FAKYaWQ6ICLHKeQMpMx%2FbnVtYmXlCQp1c2XlAgDqA6rGVHVzZUNvdeQKY2ludDY02UDkBJRzyz7lBI%2FuA%2BbGDs5O6wPg7AK%2BxFLGSHPvAY%2FMJuoI%2FuwG9%2BsBeMwt8QF7zB0ncyB1bmlxdeQDZ%2F8BTPYDns9N6gld%2BwQWzzRmdWxsIGFkZHJlc%2BUA9SDIDf8DyeUDGk9wZXLlDYv4Cv1hbGlhc%2BgMmuYOPyA9IOcMu3NSZXBlYXRhYmxl5wPDcyAm5Qjyxh9Db25kacReYWzWIGxp5QMXxRtJZDvIb%2BsAlj3sDr%2FoA0HKIDztAJo%2BOwrxD0zkDzfJOTvnDjTHB%2BYCBUZvb0JvZHnpDsjuAa%2FIFOgCS%2BQF7sgexx195AFhzVBSZXN1bOYE0Mk%2BdWztBaXJO%2BsKacgu5QpxQ3VzdG9tTMYdZeYEF8gBxTToClcgRG9uZXpv6A8QyCDmCkggQm9ya%2BQKGs4d6AoyIENodWPPIEhhdmVBbm90aOQKC8gV5A8xxAHHB%2BQFAnJm5A3%2FRm9v6wCbQOUHq2FyKOYH%2FuoA8SwgcG9sbMgOKSA6IHvGIzrwAOl95QKwx0%2FENWluZ%2BkB8ShGb28uYmFyLCB7xlDnAj3lCETkCIU8IuwBqSI%2BfeYO8sZNdeQGlm8oQGLkAe%2FEBTroAf0pxQrmAbfrAZwK&e=%40azure-tools%2Ftypespec-autorest&options=%7B%7D).

Maybe not relevant to this pr, I notice that we could customize lro status and I was wondering if Java is supported or not?

```
enum CustomLroStates {
          @lroSucceeded Donezo,
          @lroFailed Borked,
          @lroCanceled Chucked,
          HaveAnother,
        }
```

Reply:
I don't have example on parameter mapping, you had to ask Mark C. https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/decorators#@Azure.Core.operationLink

Reply:
Java's status is extensible enum https://learn.microsoft.com/en-us/java/api/com.azure.core.util.polling.longrunningoperationstatus?view=azure-java-stable